prelude/slang-cuda-prelude.h

```diff
@@ -131,67 +131,113 @@ union Union64
 // ----------------------------- F32 -----------------------------------------
 
 // Unary 
-SLANG_CUDA_CALL float F32_rcp(float f) { return 1.0f / f; }
+SLANG_CUDA_CALL float F32_ceil(float f) { return ::ceilf(f); }
+SLANG_CUDA_CALL float F32_floor(float f) { return ::floorf(f); }
+SLANG_CUDA_CALL float F32_round(float f) { return ::roundf(f); }
+SLANG_CUDA_CALL float F32_sin(float f) { return ::sinf(f); }
+SLANG_CUDA_CALL float F32_cos(float f) { return ::cosf(f); }
+SLANG_CUDA_CALL void F32_sincos(float f, float& s, float& c) { ::sincosf(f, &s, &c); }
```

Comment:
Oh, whoops. Yeah, I slotted `sincos` in after `sin` and `cos` because of semantic similarity. I might actually argue for refactoring these declarations to be grouped by the semantics of the different functions rather than by arity.